code/game/dna/genes/disabilities.dm

```diff
@@ -116,7 +112,7 @@
 			if(1)
 				H.emote("twitch")
 			if(2 to 3)
-				H.say("[prob(50) ? ";" : ""][pick("SHIT", "PISS", "FUCK", "CUNT", "COCKSUCKER", "MOTHERFUCKER", "TITS")]")
+				H.say("[prob(50) ? ";" : ""][pick("SHIT", "PISS", "FUCK ME SIDEWAYS", "VULVA VIBRATING A VENEZULEAN VUVUZELA", "PREHENSILE PENISES", "MOTHERFUCKER", "TITS")]")
```

Comment:
PFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF THIS WAS NOT SUPPOSED TO BE COMITTED. 👀